### PR TITLE
Sarcina practica nr 8

### PR DESCRIPTION
**Pull request by AnatolieQACurs**

### CODE REVIEW

File: src/com/company/topic3/ExecuteOperations.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.company.topic3;
+
+
+public class ExecuteOperations {
+    public static void main(String[] args) {
+        Operations.printLogicOperations();
```
Comment (AndreiSecu1997) [inline src/com/company/topic3/ExecuteOperations.java:6]: remove comments before commit please

File: src/com/company/topic3/Operations.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+               System.out.print(num + " ");
+               num++;
+           }
+            System.out.println(" While");
+
+               for (int i = 5;  i <= lim; i++){
+                   System.out.print(i + " ");
+               }
+               System.out.println(" For ");
+
```
Comment (AndreiSecu1997) [inline src/com/company/topic3/Operations.java:44]: empty lines

File: src/com/company/topic3/ExecuteOperations.java (hunk 1 above)
Comment (AnatolieQACurs) [reply src/com/company/topic3/ExecuteOperations.java:6]: I have modified you can check again

File: src/com/company/topic3/Operations.java (hunk 2 above)
Comment (AnatolieQACurs) [reply src/com/company/topic3/Operations.java:44]: ok

**Pull request merged**